node/status_node_services.go

```diff
@@ -136,7 +136,7 @@ func (b *StatusNode) initServices(config *params.NodeConfig, mediaServer *server
 		if len(openseaKey) == 0 {
 			openseaKey = OpenseaKeyFromEnv
 		}
-		walletService := b.walletService(accDB, accountsFeed, openseaKey)
+		walletService := b.walletService(accDB, accountsFeed, openseaKey, config.WalletConfig.AlchemyAPIKeys, config.WalletConfig.InfuraAPIKey, config.WalletConfig.InfuraAPIKeySecret)
```

Comment:
Maybe it's better to pass config.WalletConfig ?

Reply:
Yeap... The `OpenseaKeyFromEnv` thing was needed before the Mobile client passed the key through the config, but it's doing it now so I'll remove that and pass the struct to make it cleaner

Reply:
Done